.claude/skills/observability-instrumentation/reference/metrics-patterns.md

```
@@ -0,0 +1,607 @@
+            $duration = (microtime(true) - $startTime) * 1000;
+            $this->metrics->record('mongodb.customer.save.duration', $duration, ['status' => 'error']);
+            $this->metrics->increment('mongodb.customer.save.errors', ['error_type' => get_class($e)]);
+            throw $e;
+        }
+    }
+
+    public function findById(CustomerId $id): ?Customer
+    {
+        $startTime = microtime(true);
+
+        try {
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)